workshop/modules/terraform-aws-eks-data-addons/values/emr-spark-operator-values.yaml

```diff
@@ -0,0 +1,29 @@
+replicaCount: 1
+
+webhook:
+  # -- Enable webhook server
+  enable: true
+  # -- Webhook service port
+  port: 8080
+
+emrContainers:
```

Comment:
Set only this value in this module and ignore the other values as those are the default